docs/introduction/composition.md

```diff
@@ -819,4 +819,4 @@ known issues and proposed improvements.
 [Infrastructure Composition Provisioning]: composition-provisioning.png
 [composition related issues]: https://github.com/crossplane/crossplane/labels/composition
 [#1481]: https://github.com/crossplane/crossplane/issues/1481
-[#1909]: https://github.com/crossplane/crossplane/issues/1909
\ No newline at end of file
+[#1909]: https://github.com/crossplane/crossplane/issues/1909
```

Comment:
This looks like a newline modification, potentially unintentional?

Reply:
I don't know why it adds this row!

Reply:
It is likely just a setting in your editor, no worries if it hangs around :) I actually couldn't tell if it was adding or removing a newline, we generally prefer to have a newline so maybe it is actually fixing a missing one :+1: